js/index.js

```diff
@@ -0,0 +1,61 @@
+$(document).ready(function() {
+	//First we'll need our variables
+	var answer; //This will be the randomly generated number
+	//answer = Math.floor((Math.random() * 100) +1); 
+	//answer = 5;
+	var guess; //This will be the user's current guess
+	var count = 5; //This is the number of guesses remaining
+	//$('#reset').click(resetFunc);
+
+	$('button').click(function(event) {
+		event.preventDefault();
+            var button_type = ($(this).attr("value"));
```

Comment:
@filippia Not sure what's going on with line 12 here. What were you trying to achieve? Perhaps we could review the code together.